src/vue/components/ArticleActions.vue

```diff
@@ -1,17 +1,17 @@
 <template>
   <!-- Used when user is also author -->
   <span v-if="canModify">
-    <router-link class="btn btn-sm btn-outline-secondary" :to="editArticleLink">
+    <router-link data-cy="edit-article-link" class="btn btn-sm btn-outline-secondary" :to="editArticleLink">
```

Comment:
for better readability think about using the feature name as a prefix (not required to fix, just a recommendation)